dashbuilder/image.yaml

```diff
@@ -0,0 +1,157 @@
+    description: "If true components will be partitioned by the Runtime Model ID. Sets the dashbuilder.components.partition property. Default value is false."
+  - name: "DASHBUILDER_DATASET_PARTITION"
+    example: "true"
+    description: "If true datasets IDs will partitioned by the Runtime Model ID.. Sets the dashbuilder.dataset.partition property. Default value is false."
+  - name: "DASHBUILDER_IMPORT_FILE_LOCATION"
+    example: "/opt/mypv/my_static_dsahboard.zip"
+    description: "Set a static dashboard to run with runtime. When this property is set no new imports are allowed. If provided file is not set, Dashbuilder image will fail to start."
+  - name: "DASHBUILDER_IMPORTS_BASE_DIR"
+    example: "/path/to/my/pv/with/dashboards"
+    description: "Base Directory where dashboards ZIPs are stored, if not set or if the provided value does not exist, the '/opt/kie/data/imports' will be used."
+  - name: "DASHBUILDER_MODEL_UPDATE"
+    example: "false"
+    description: "Boolean property that allows Runtime to check model last update in FS to update its content. Sets the dashbuilder.model.update property. Default value is true."
+  - name: "DASHBUILDER_MODEL_FILE_REMOVAL"
+    example: "true"
+    description: "Boolean property when true will also remove actual model file from file system. Sets the dashbuilder.removeModelFile property"
+  - name: "DASHBUILDER_RUNTIME_MULTIPLE_IMPORT"
+    example: true
+    description: "If set to true Runtime will always allow use of new imports (multi tenancy). Sets the dashbuilder.runtime.multi property. Default value is false"
+  - name: "DASHBUILDER_UPLOAD_SIZE"
+    example: "10000000"
+    description: "Limits the size of uploaded dashboards (in kb). Sets the dashbuilder.runtime.upload.size. Default value is 10485760 kb."
+  - name: "DASHBUILDER_COMP_ENABLE"
+    example: true
+    description: "Boolean property that when true enables external components. Sets the dashbuilder.components.enable property. Default value is false"
+  - name: "DASHBUILDER_EXTERNAL_COMP_DIR"
+    example: "/tmp/test"
+    description: "Configures the base directory for dashbuilder components. Sets the dashbuilder.components.dir property. Default value is /opt/kie/components"
+  - name: "DASHBUILDER_CONFIG_MAP_PROPS"
+    example: "/data/config.properties"
+    description: "Properties file with Dashbuilder configurations, if set, uniq properties will be appended and, if a property is set mode than once, the one from this property file will be used."
+  - name: "KIESERVER_DATASETS"
+    example: "dataset_1,dataset_2"
+    description: "Configures the KIE Server Datasets prefix. For multi datasets, use comma separated values. The valid values for this prefix are: ${datasetPrefix}_LOCATION, ${datasetPrefix}_USER, ${datasetPrefix}_PASSWORD, ${datasetPrefix}_TOKEN. For more information about these properties please take a look on https://docs.jboss.org/jbpm/release/7.45.0.Final/jbpm-docs/html_single/#_kie_server_integration"
```

Comment:
I see it is missing `replace_query` - which is used to have dashbuilder creating and replacing queries on Kie Server. Default value is false.

https://github.com/kiegroup/appformer/blob/master/dashbuilder/dashbuilder-backend/dashbuilder-kie-server-backend/src/main/java/org/dashbuilder/kieserver/backend/KieServerConnectionInfoProviderImpl.java#L62